app/helpers/helpers.rb

```diff
@@ -0,0 +1,10 @@
+class Helpers
```

Comment:
This isn't the correct `helpers` pattern, see: http://www.sinatrarb.com/intro.html#Helpers